internal/telemetry/event.go

```diff
@@ -55,11 +56,17 @@ func withProfile() eventOpt { // either "default" or base64 hash
 
 func withPrompt(p, k string) eventOpt {
 	return func(event Event) {
-		event.Properties["prompt"] = p
+		event.Properties["prompt"] = sanitizePrompt(p)
 		event.Properties["prompt_type"] = k
 	}
 }
 
+func sanitizePrompt(q string) string {
```

Comment:
With this, we should probably update our wiki to reinforce that every sensitive info is displayed on brackets. Is there any way for us to automate this behaviour? Not blocking this PR, just throwing my concern for us to discuss.

Reply:
Those are just default settings that are displayed at the start - there is no user interaction and nothing that we are required to track in that regard.